src/common/envalid.ts

```diff
@@ -13,6 +13,7 @@ const cleanSpec = {
   OTOMI_DEV: bool({ default: false }),
   OTOMI_IN_TERMINAL: bool({ default: true }),
   STATIC_COLORS: bool({ default: false }),
+  TEAM_IDS: json({ desc: 'A list of team ids in JSON format' }),
```

Comment:
Can we maybe make the user experience a bit less painful and provide a default value here?

Reply:
Good point. Still it does not feel right to extend possible env list that are used by CLI

Reply:
No. Default values exist for sanity. It would be insane to presume to know anything about user input

Reply:
And what you mean CLI only? This is business/config/topology for software. It has nothing to do with context it is used in

Reply:
> No. Default values exist for sanity.

Assumption/your opinion.

> It would be insane to presume to know anything about user input

Not at all, you should think more about how a user with less knowledge than you perceives it.

Reply:
> > No. Default values exist for sanity.
> 
> Assumption/your opinion.

Really? Defaults are assumptions about what the majority of users want.

> > It would be insane to presume to know anything about user input
> 
> Not at all, in my opinion you should think more about how a user with less knowledge than you perceives it.

Ok, I did. Did you? Can you provide defaults for the names of teams end users will want to provide? No. Please help by giving arguments instead of negations and general remarks.

Reply:
Can you please stop trying to win an argument and just focus on improving the code? You never try to get the perspective of other people.

If we take a random application like Jenkins or Kubernetes, it does not expect me to provide a list of user id's I want to see created at the start-up. It just gives the default user e.g. jenkins or admin. So in my opinion we should just provide the user with a team that can be deleted afterwards if the name is not appropriate, e.g. team otomi.

Reply:
That is not I as a business owner want to make. We see a different path then you (like seeing the added value of a user point that themselves and learning from the process). Please ask if you are interested in UX decisions instead of stating you have a "better" UX decision.